reduction_scripts/reduce_data.py

```diff
@@ -36,24 +36,18 @@
     else:
         obs_mode = 'class'    
 
-    #************************************************************************
-    #*****       LOAD JSON FILE WITH USER REDUCTION SETUP              *****
-    #************************************************************************
     # Read the JSON file
-    file_path = 'params_'+obs_mode+'.json'
+    file_path = f'params_{obs_mode}.json'  
     with open(file_path, 'r') as f:
         proc_steps = json.load(f)
-    #************************************************************************
-
 
     # Check if the reduc_blue/red folders already exists and create them if required
-    out_dir = os.path.join(proj_dir, 'reduc_'+arm) 
+    out_dir = os.path.join(project_dir, f'reduc_{arm}') 
     if not os.path.exists(out_dir):
         os.mkdir(out_dir)
     else:
-        print(f"Folder '{'reduc_'+arm}' already exists.")
+        out_dir: print(f"Folder '{out_dir}' already exists.")
```

Comment:
I don't think you need `out_dir: ` at the start of the line here.

Reply:
Yes, true... I realised afterward. Is fixed now.